src/resources/loadresources.cpp

```diff
@@ -0,0 +1,38 @@
+#include "loadresources.h"
+
+#include <qglobal.h>
+
+using namespace OCC;
+
+void static load_rc()
+{
+    Q_INIT_RESOURCE(ownCloudResources_translations);
```

Comment:
According to the Qt docs, we should not need any of these calls when using a shared library, it should be handled by Qt itself. I assume you have tested this without these calls? I do not have any experience moving Qt resources into a dedicated shared library, so I'm curious.

https://doc.qt.io/qt-6/resources.html

Reply:
Hm I haven't tested it, however we would still need to export a and use symbols from this lib.